modules/nodes-pods-autoscaling-more-about.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+:_content-type: CONCEPT
+[id="nodes-pods-autoscaling-more-about_{context}"]
+= More about horizontal pod autoscalers
+
+The horizontal pod autoscaler (HPA) extends the concept of pod auto-scaling. The HPA lets you create and manage a group of load-balanced nodes. The HPA automatically increases or decreases the number of pods when a given CPU or memory threshold is crossed.
```

Comment:
Could this paragraph just be at the start of the "How does the HPA work" section? Versus having a separate "More about..." heading, considering there's already an "Understanding..." heading right before it.

E.g., with the TOC ultimately looking something like:

* Understanding horizontal pod autoscalers
  *	How does the HPA work?
  *	Supported metrics
  *	About requests and limits
  *	How to use resource metrics?
  *	Best practices
  *	Scaling practices

You'd probably want to break this up into separate modules to achieve that, though.

Reply:
Going with your suggestion.
I have kept About requests and limits and How to use resource metrics? as one conceptual module as they are interelated.

Reply:
Supported Metrics is part of the Understanding horizontal pod auto scaler module.  So the TOC would look like:

Understanding horizontal pod autoscalers (part of Supported metrics)
How does the HPA work?
About requests and limits (including how to use resource metrics?)
Best practices
Scaling practices